### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @santismith

### CODE REVIEW

**Comment by santismith** [comment]
terminé @faloi

File: README.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ En ambos casos, incluir solamente los métodos y atributos que consideren import
 
 Y de yapa, dos ejemplos de cómo incluir una imagen en Markdown (vean este archivo desde un editor para notar la diferencia):
 
-![Directamente desde un link](http://www.plantuml.com/plantuml/png/JSqn2W9138NXFgT8gLUmMb5hlS9Z9gu3cGmaMH74kxl25lR_zzuK2S-bAnL51DyKC_XDpEcmw2tbOdVnBXLrjp_mKUT17zhoqrBWA0y4ool5wl-2jQVe6TR63rxJnuQdDLmllm00)
 
+DIRECTAMENTE DESDE EL LINK :
+![Directamente desde un link](http://www.plantuml.com/plantuml/png/bPBTJiCm38Nl-nIvR5zXLzluOPCegWGlu2LU2KhZuJW8QUFT2LEMqQeIuhAHp-TpZhiC0YnnS9Ls489Q0xCbTQoKGeS3Ug6W5khHfhdggyfJdOCUpaC7sZdGbdmQsdf9ByBWms05x_mUUK03fcuMQata47mQ004DN5_mZLhK6-g1Joq4JGAdJKGMQ2RhRc9IWNu5XP-2tenSOA2EAIspL8nu-u7k5hobkicFOs9BuJrQeHdWdzekkXx2M3aLpkaUW1aE8ECK6NUyjlCh_RcSmcvZprWoQl9xOYdV6DOHFOqBtqTS_L0DbTAdtdBrr-pFuAB9r5d0pChBnglpY9oXEeNMre3fc4pK-O-vTmJ_ivrTq-uDjQZjqylgVFy2RQst0vIJBT6REBXl)
```
Comment (faloi) [inline README.md:25]: El `Barrio` te quedó flaquito, sin métodos. El resto ok.
Banco el trabajo en equipo, pero sería bueno que cuando el laburo es individual se respete.

File: src/barrio.wlk [hunk 2]
```diff
@@ -1,24 +1,31 @@
 class Barrio {
 
+    // se crea la lista
```
Comment (faloi) [inline src/barrio.wlk:3]: ¿Lista de qué tipo?

File: src/barrio.wlk [hunk 3]
```diff
@@ -1,24 +1,31 @@
...
 	const property elementos = []
 
+
+    //agrega un objeto(tipo any),que recibe por parametro, a la lista. No devuelve nada
 	method agregarElemento(elemento) {
 		elementos.add(elemento)
 	}
 
+    //elimina un objeto el cual se recibe por parametro que se encuentra en la lista. No devuelve nada
```
Comment (faloi) [inline src/barrio.wlk:12]: Ojo, no interesa que expliques qué hace el método, solo pedimos que pongas el tipo.

File: src/elementos.wlk [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
 
 	// Se considera que un hogar _es bueno_ 
-	// si su nivel de mugre es la mitad del confort que ofrece, o menos. 
+	// si su nivel de mugre es la mitad del confort que ofrece, o menos.
+
+	//metodo que devuelve un booleano, preguntando si el nivel de mugre es menor o igual que el confort dividido por 2
 	method esBueno() = nivelMugre <= confort / 2
 
+
+    //metodo que no devuelve nada,recibe un objeto tipo plaga(cucaracha,mosquito,garrapata,pulga)por parametro,
```
Comment (faloi) [inline src/elementos.wlk:14]: Me gustó que escribiste "tipo **plaga**", es exactamente ese el tipo de este parámetro.

File: src/elementos.wlk [hunk 5]
```diff
@@ -15,26 +21,35 @@ class Hogar {
...
+    //metodo que devuelve un booleano, pregunta si el nivel de salud es mayor a 250, caso contrario devuelve False.
 	method esBueno() = nivelSalud > 250
 
 }
 
 class Huerta {
 
-	var property capacidadProduccion
 
+	var property capacidadProduccion //de tipo entero o real
```
Comment (faloi) [inline src/elementos.wlk:43]: Je, buena observación la de _real_.

**Pull request closed without merging**